### PR TITLE
barracuda firewall documentation

### PR DESCRIPTION
**Pull request by lzammit**

### CODE REVIEW

File: docs/PacketFence_Barracuda_Quick_Install_Guide.asciidoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* *Firewall type: Barracuda (Barracuda = SSH requests)
+* *Password: secret 
+* *Port: 22
+* *Roles: add the roles that you want to do SSO
+
+image::docs/images/doc-barracuda-cfg_SSO_pf.png[scaledwidth="100%",alt="Create the SSO agent in Barracuda"]
+
+Step 2: Verification
+~~~~~~~~~~~~~~~~~~~~
+Try to do a connection to packetfence, if you user is match a correct rule with the role that you have configured, the packetfence server will send a request to the Barracuda to allow you.
```
Comment (extrafu) [inline docs/PacketFence_Barracuda_Quick_Install_Guide.asciidoc]: Reformulate entirely. It's not clear.

packetfence -> PacketFence.

File: docs/PacketFence_Barracuda_Quick_Install_Guide.asciidoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+If you want to see if it's working, you can log into the Barracuda in SSH and run this command following commands: acpfctrl auth show
```
Comment (extrafu) [inline docs/PacketFence_Barracuda_Quick_Install_Guide.asciidoc]: in SSH -> user SSH

Comment (extrafu) [inline docs/PacketFence_Barracuda_Quick_Install_Guide.asciidoc]: this following command: ... put the command as a code block.

**Comment by julsemaan** [comment]
Missing the docbook, makefile and in packetfence.spec

**Pull request merged**